eva/executor/create_executor.py

```diff
@@ -29,36 +29,8 @@ def validate(self):
         pass
 
     def exec(self):
-        """Create table executor
-
-        Calls the catalog to create metadata corresponding to the table.
-        Calls the storage to create a spark dataframe from the metadata object.
-        """
-
-        if (self.node.if_not_exists):
-            
-            # check if the table exists
-            catalog_manager = CatalogManager()
-
-            # TODO: Due to the way get_table_bindings is written, 
-            # an error will be logged here if the table does not exist. 
-            # But that is okay since we are just checking if the table exists. 
-            table_name = self.node.video_ref.table_name
-            metadata_id, _ = catalog_manager.get_table_bindings("eva_catalog",
-                                                                table_name)
-
-            # metadata_id is not None when the table exists, so return
-            if metadata_id is not None:
-                return
-
-        # TODO: I had to comment out the following line,
-        # because it was causing an error. table_name is in video_ref, 
-        # and not in table_info.
-        # table_name = self.node.video_ref.table_info.table_name
-        table_name = self.node.video_ref.table_name
-        file_url = str(generate_file_path(table_name))
-        metadata = CatalogManager().create_metadata(table_name,
-                                                    file_url,
-                                                    self.node.column_list)
-
-        StorageEngine.create(table=metadata)
+        if not check_table_exists(self.node.table_ref,
+                                  self.node.if_not_exists):
```

Comment:
`check_table_exists(self, database_name: str, table_name: str)`. The second argument is `table_name` instead of `self.node.if_not_exists`. Need to have a if for `self.node.if_not.exists`?